### PR TITLE
Add `# noqa` comments to `AnyStr`

### PR DESCRIPTION
**Pull request by AlexWaygood**
Unblocks https://github.com/PyCQA/flake8-pyi/pull/79

### CODE REVIEW

File: stdlib/@python2/typing.pyi [hunk 1]
```diff
@@ -77,7 +77,7 @@ Counter = _Alias()
 Deque = _Alias()
 
 # Predefined type variables.
-AnyStr = TypeVar("AnyStr", str, unicode)
+AnyStr = TypeVar("AnyStr", str, unicode)  # noqa Y001
```
Comment (Akuli) [inline stdlib/@python2/typing.pyi:80]:
```suggestion
AnyStr = TypeVar("AnyStr", str, unicode)  # noqa: Y001
```

File: stdlib/typing.pyi [hunk 2]
```diff
@@ -136,7 +136,7 @@ if sys.version_info >= (3, 9):
     Annotated: _SpecialForm = ...
 
 # Predefined type variables.
-AnyStr = TypeVar("AnyStr", str, bytes)
+AnyStr = TypeVar("AnyStr", str, bytes)  # noqa Y001
```
Comment (Akuli) [inline stdlib/typing.pyi:139]:
```suggestion
AnyStr = TypeVar("AnyStr", str, bytes)  # noqa: Y001
```

**Review by Akuli — APPROVED** [review]

File: stdlib/@python2/typing.pyi (hunk 1 above)
Comment (AlexWaygood) [reply stdlib/@python2/typing.pyi]: Heh, I actually just copied and pasted the `# noqa` comment from `_typeshed.Self`

Comment (Akuli) [reply stdlib/@python2/typing.pyi]: Maybe it works with both.

**Pull request merged**